lib/elixir/test/elixir/enum_test.exs

```diff
@@ -47,6 +48,14 @@ defmodule EnumTest do
     assert Enum.at([2, 4, 6], -4) == nil
   end
 
+  test "chunk/3" do
+    enumerable = [1, 2, 3, 4, 5]
+
+    capture_io(fn ->
```

Comment:
So the warning is emitted at compilation time, which means `capture_io` does not really do anything here? 🤔

Reply:
So you are not seeing the warning because of a bug in your deprecation checker but here is what you need to do:

```
enum = Enum
assert enum.chunk(enumerable, 2, 1)
```

and that should be warning free. No need for `capture_io`.

Reply:
When you said that … However in my case I do not see warnings when using `capture_io/1` (commenting that call shows warning). Probably because it's not working strictly like `mix test`, but there is `elixir` binary called with `-r` and `-pr` arguments in `Makefile`, right?

Reply:
I have fixed the bug in master, so you should see the deprecation warnings now. :)